lib/Serialization/SourceInfoFormat.h

```diff
@@ -0,0 +1,94 @@
+//===--- SourceInfoFormat.h - Format swiftsourceinfo files ---*- c++ -*---===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+///
+/// \file Contains various constants and helper types to deal with serialized
+/// source information (.swiftsourceinfo files).
+///
+//===----------------------------------------------------------------------===//
+
+#ifndef SWIFT_SERIALIZATION_SOURCEINFOFORMAT_H
+#define SWIFT_SERIALIZATION_SOURCEINFOFORMAT_H
+
+#include "llvm/Bitcode/RecordLayout.h"
+
+namespace swift {
+namespace serialization {
+
+using llvm::BCArray;
+using llvm::BCBlob;
+using llvm::BCFixed;
+using llvm::BCGenericRecordLayout;
+using llvm::BCRecordLayout;
+using llvm::BCVBR;
+
+/// Magic number for serialized source info files.
+const unsigned char SWIFTSOURCEINFO_SIGNATURE[] = { 0xD6, 0x9C, 0xB7, 0x23 };
```

Comment:
Where does this number come from? :-)

Reply:
I made these number up. Is there a recommended way?

Reply:
Concept: use the UTF8 bytes of 👩‍⚕️

Reply:
Concept: use the UTF8 bytes of 📒

Reply:
I end up using 🏎️as the magic number :)